### ISSUE TITLE
Minify current file and output directly

### ISSUE THREAD

**Issue by Morgy93** [HookyQR/VSCodeMinify#14]
Hello,

would it be possible to just minify the current opened file?
Just overwrite the current opened text?
Even better would be like most other extensions do with some preview button, to have another panel with the minified code in it.

**Comment by anyway** [comment]
This would be particularly useful for HTML files. Currently the plugin creates a `*.min.html`. So we need to delete the `*.html` and rename the `*.min.html` to `*.html` before we can upload it.

**Comment by CharlBest** [comment]
This would be very nice to have
